src/cascadia/TerminalApp/TerminalApp.vcxproj

```diff
@@ -86,7 +86,7 @@
       <AdditionalIncludeDirectories>$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>User32.lib;WindowsApp.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>$(_BinRoot)\WinRTUtils\WinRTUtils.lib;User32.lib;WindowsApp.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
This is caused by VS not being 16.3.9. We probably don't need it now

Reply:
Yep, looks fine after I updated. Thanks.